docs/guides/modules/persistence.rst

```diff
@@ -73,17 +81,42 @@ attribute on our object. Let's include the author's name as an attribute of a bo
     assert book.author_name == "Luciano Ramalho" 
 
 
+LinkedClass
++++++++++++
+Alternatively, we can define another class for author objects and use an instance of that
+class as an attribute of a Book:
+
+.. code-block:: python
+
+   from anvil_labs.persistence import persisted_class, LinkedClass
+
+   @persisted_class
+   class Author:
+       pass
+
+   @persisted_class
+   class Book:
+       author = LinkedClass(linked_colum="author", constructor=Author.create)
```

Comment:
I'd almost want the API to infer linked_column was `"author"` unless I specified it as something else.

And looking at the tests, should the user be calling the `.create()` method themselves?

I guess the user has to know a `.create` method gets added in this version of the API.
I think I'd prefer an interface where the LinkedClass has to be a `persisted_class` and all they have to do is provide it.


```python

   @persisted_class
   class Book:
       author = LinkedClass(Author)

```

Which also implies I might be able to just do

```python

   @persisted_class
   class Book:
       author = Author

```


Good luck with circular links 😅

Reply:
> I'd almost want the API to infer linked_column was "author" unless I specified it as something else.

Yep, that's what I had in mind to have a go at next

> And looking at the tests, should the user be calling the .create() method themselves?

Yes. The reasoning here was to avoid using `__init__` ourselves so that, if users create their own, we don't have to sort out handling inheritance

> I think I'd prefer an interface where the LinkedClass has to be a persisted_class and all they have to do is provide it.

Nice idea - I'll have a go at that.